src/Sarif.Multitool.FunctionalTests/TestData/UriBaseIdRequiresRelativeUri/UrisAreRelative.sarif

```diff
@@ -1,52 +1,204 @@
+                  "uri": "/c:/src/file.c"
+                }
+              }
+            }
+          ]
+        }
+      ],
       "results": [
         {
           "ruleId": "TST0001",
           "level": "error",
-          "formattedRuleMessage": {
-            "formatId": "Default"
+          "analysisTarget": {
+            "uriBaseId": "%SRCROOT%",
+            "uri": "/c:/src/file.c"
           },
-          "locations": [
+          "attachments": [
+            {
+              "fileLocation": {
+                "uri": "/c:/src/file.c",
+                "uriBaseId": "%SRCROOT%"
+              }
+            }
+          ],
+          "conversionProvenance": [
             {
-              "analysisTarget": {
-              "uriBaseId": "%SRCROOT%",
+              "fileLocation": {
                 "uri": "/c:/src/file.c",
-                "region": {
-                  "startLine": 1
+                "uriBaseId": "%SRCROOT%"
+              }
+            }
+          ],
+          "graphs": [
+            {
+              "id": "g1",
+              "nodes": [
+                {
+                  "id": "n1",
+                  "location": {
+                    "physicalLocation": {
+                      "fileLocation": {
+                        "uri": "/c:/src/file.c",
+                        "uriBaseId": "%SRCROOT%"
+                      }
+                    }
+                  }
                 }
-              },
+              ],
+              "edges": [
+                {
+                  "id": "e1",
+                  "sourceNodeId": "n1",
+                  "targetNodeId": "n1"
+                }
+              ]
+            }
+          ],
+          "locations": [
+            {
               "physicalLocation": {
-                "uriBaseId": "%SRCROOT%",
-                "uri": "/c:/src/file2.c",
-                "region": {
-                  "startLine": 1
+                "fileLocation": {
+                  "uri": "/c:/src/file.c",
+                  "uriBaseId": "%SRCROOT%"
                 }
               }
             }
           ],
           "relatedLocations": [
             {
               "physicalLocation": {
-                "uriBaseId": "%SRCROOT%",
-                "uri": "/c:/src/file3.c"
+                "fileLocation": {
+                  "uriBaseId": "%SRCROOT%",
+                  "uri": "/c:/src/file.c"
+                }
               }
             }
           ],
           "codeFlows": [
             {
-              "locations": [
+              "threadFlows": [
                 {
-                  "physicalLocation": {
-                    "uriBaseId": "%SRCROOT%",
-                    "uri": "/c:/src/file1.c"
-                  }
+                  "locations": [
+                    {
+                      "location": {
+                        "physicalLocation": {
+                          "fileLocation": {
+                            "uriBaseId": "%SRCROOT%",
```

Comment:
>SRCROOT [](start = 43, length = 7)

UBER NIT: SRCROOT doesn't seem like the right variable name for a relative path that starts with a drive letter. :) No change proposed, just commenting to amuse myself.

Reply:
Oh, good point. I think I'll fix it; it will just make the files that much shorter.

---
In reply to: [210453175](https://github.com/Microsoft/sarif-sdk/pull/1018#discussion_r210453175) [](ancestors = 210453175)